src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/pci-dss.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: PCI DSS
+tags:
+  - Security
+  - Security and Privacy
+  - Compliance
+metaDescription: Criteria and compliance with PCI DSS audits by New Relic services.
+freshnessValidatedDate: 2024-07-25
+redirects:
+  - /docs/security/security-privacy/compliance/certificates-standards-regulations/PCI DSS-moderate
+---
+
+The Payment Card Industry Data Security Standard (“PCI DSS”), maintained by the Security Standards Council, is a set of security requirements to protect cardholder data environments (“CDEs”) where payment card data is stored, processed, or transmitted. PCI DSS provides a baseline of rigorous technical and operational requirements designed to protect CDEs.
```

Comment:
You cannot use an article here ... The Payment Card Industry Data Security Standard